styles.scss

```diff
@@ -549,7 +549,9 @@ blockquote {
   display: grid;
   grid-gap: 10px;
```

Comment:
In my tests in the devtools, `grid-gap: 7px 10px` gives slightly better results with the added transport info height.